pkg/kube/wrappers/ingress.go

```diff
@@ -70,8 +70,11 @@ func (iw *IngressWrapper) GetURL() string {
 	// Append port + ingressSubPath
 	url += iw.getIngressSubPath()
 
-	return url
+	return url + iw.getAppRoot()
```

Comment:
I think there must be a check to make sure that the user doesn't provide an invalid value. 

For example:
Example 1:

url = "xyz.com/"
appRoot: "/myApp"

In this case, an invalid URL path will  be generated containing two `//` in it

Example 2:
url = "xyz.com"
appRoot = "myApp"

In this case, an invalid URL path will be generated containing no separation due to /.

There are two ways to solve this problem either by adding checks in the code or by adding a CAVEAT in the README.md file explicitly specifying where to add the `/`.

Reply:
I have made changes to the PR and this is no longer valid